plutus-conformance/src/PlutusConformance/Common.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{- | Plutus conformance test suite library. -}
+module PlutusConformance.Common where
+
+import Control.Exception (SomeException, evaluate, try)
+import Data.Text qualified as T
+import Data.Text.IO qualified as T
+import PlutusCore.Core (defaultVersion)
+import PlutusCore.Default (DefaultFun, DefaultUni)
+import PlutusCore.Error (ParserErrorBundle (ParseErrorB))
+import PlutusCore.Evaluation.Machine.ExBudgetingDefaults (defaultCekParameters)
+import PlutusCore.Evaluation.Result (EvaluationResult (..))
+import PlutusCore.Name (Name)
+import PlutusCore.Quote (runQuoteT)
+import PlutusPrelude
+import Test.Tasty (TestTree, defaultMain, testGroup)
+import Test.Tasty.Golden (findByExtension)
+import Test.Tasty.HUnit (testCase, (@=?))
+import Text.Megaparsec (SourcePos)
+import UntypedPlutusCore.Core.Type qualified as UPLC
+import UntypedPlutusCore.Evaluation.Machine.Cek (unsafeEvaluateCekNoEmit)
+import UntypedPlutusCore.Parser qualified as UPLC
+
+-- | A TestContent contains what you need to run a test.
+data TestContent =
+   MkTestContent {
+       -- | The path of the input file. This is also used to name the test.
+       testName    :: FilePath
+       -- | The expected output of the test in `Text`.
+       , expected  :: T.Text
+       -- | The input to be tested, in `Text`.
+       , inputProg :: T.Text
```

Comment:
If you make this an actual test you might want to indeed have these be file paths and do the reading inside the test itself, just so the IO is all in one place.

Reply:
Yes so I got them as file paths so I don't see a real need to turn them into `Text`.